spec/jobs/catalog_job_spec.rb

```diff
@@ -8,6 +8,10 @@ def update_registry!(catalog)
 end
 
 describe CatalogJob do
+  before :each do
+    VCR.turn_on!
```

Comment:
Good question. All I know is that this gives us 100% coverage. Maybe that's just a happy little coincidence.